src/components/MainHeader/MainHeaderStyles.ts

```diff
@@ -11,6 +11,13 @@ const StyledMainHeader = styled.header`
   justify-content: space-between;
   padding: 0 1rem;
   background-color: #fff;
+  font-size: 3.5rem;
+  font-family: "Rubik Puddles", cursive;
+  word-spacing: 0.5rem;
+
+  & img {
+    height: 3rem;
```

Comment:
it's almost never a good idea to set a rigid height value. It can hinder your responsiveness and other layouts. (There are some exceptions obviously, just try to make sure it works well first.)